Libraries/LibELF/Validation.cpp

```diff
@@ -193,6 +193,13 @@ bool validate_program_headers(const Elf32_Ehdr& elf_header, size_t file_size, co
 
     for (size_t header_index = 0; header_index < num_program_headers; ++header_index) {
         auto& program_header = program_header_begin[header_index];
+
+        if (program_header.p_filesz > program_header.p_memsz) {
+            if (verbose)
+                dbgprintf("Program header (%zu) has p_filesz (%u) larger than p_memsz (%u)\n", header_index, program_header.p_filesz, program_header.p_memsz);
```

Comment:
The latest and greatest would be:

```suggestion
                dbgln("Program header ({}) has p_filesz ({}) larger than p_memsz ({})", header_index, program_header.p_filesz, program_header.p_memsz);
```